tools/pylib/boutdata/data.py

```diff
@@ -247,6 +249,50 @@ def __init__(self, filename, name="root"):
 
                         section[line[:eqpos].strip()] = value
 
+    def write_new(self, filename=None, force_overwrite=False):
+        """
+        Write a new input file with the options (including any modifications
```

Comment:
For the [numpdocs style][1], this should start on the same line as the triple quotes, with an empty line between the short summary and the rest. I would also simplify the short doc to:

```python
"""Write to BOUT++ options file

By default...
```

[1]: https://numpydoc.readthedocs.io/en/latest/format.html#numpydoc-docstring-guide